composer.json

```diff
@@ -105,5 +105,10 @@
                 "https://www.drupal.org/project/menu_link_weight/issues/2995896": "https://www.drupal.org/files/issues/2018-08-29/2995896-menu-admin-per-menu-support-fixes.patch"
             }
         }
+    },
+    "config": {
```

Comment:
This might cause problems with deploy no? Have the caches cleared and we can remove this now?

Reply:
I believe its only affects CI tests. but even with this it was still getting inconsistent results when running `composer update`

Reply:
I think I'd rather the prefer controls be passed in manually than by wiring it up here. Now that some time has passed let me try without this.

Reply:
yeah i dont know why it was doing that. hours after the last merge, it still wasn't consistent. i almost wonder if a few more commits on `stanford_text_editor` would push it out far enough. :man_shrugging: